docs/UserGuide.md

```diff
@@ -277,16 +343,19 @@ E.g. 2022-10-20 and 13:30.
 Examples:
 * `edite 1 d/2021-12-21 l/Zoom` Edits the date and location of the 1st event to be `2021-12-21` and `Zoom` respectively.
 * `edite 2 n/Resume Screening t/` Edits the name of the 2nd event to be `Resume Screening` and clears all existing tags.
```

Comment:
I think this is missing an empty line after.

Reply:
Done!